### DISCUSSION TITLE
Navigation links not working when ngx-editor is displayed

### DISCUSSION THREAD

**Discussion by EtienneFK** [sibiraj-s/ngx-editor#462] [Q&A]
Hello everyone,

I'm facing a curious issue where I'm not able to interact with all my links in my app when I'm rendering the editor.
It seems that the click doesn't click at all when I click on these links...

If I place a button in the same page, outside the editor, I can click on it.
If i right-click on my links then "open in new tab", it works.

I've used the [Full Featured Editor example](https://sibiraj-s.github.io/ngx-editor/en/examples/full-featured-editor/).

All my navigation links are generated in a separate component at the top level of my app, they are using routerLink.
They work perfectly when I remove the editor from the template.

I didn't find any discussion about this behaviour, neither any thing about it in the documentation.

Am I missing something obvious?

Thanks in advance for any help.

**Comment by sibiraj-s** [comment]
The left click is supposed to do nothing. it is not configured for it. It will work how a link in contenteditable will work. You can right click to open the links in new tab.

Also, see if your issue matches this issue here https://github.com/sibiraj-s/ngx-editor/discussions/460, also check the answer there

**Comment by EtienneFK** [comment]
Hello @sibiraj-s thank you for your reply.

I think I was not clear about the issue I'm facing cause the issue your pointing me doesn't fit at all my problem.

I'm not speaking about links INTO the ngx-editor, which has perfect behaviour (I can right click then open in new tab as well).
These are my nav links, OUTSIDE the ngx-editor, that stop working when I display a ngx-editor.

Btw, thank you for your time.

**Reply by sibiraj-s** [reply]
Anything reproducible for me to check?

**Comment by ShijayHuo** [comment]
Faced with the same problem, the link cannot be clicked in non-editing mode
